packages/atomic/package.json

```diff
@@ -15,7 +15,7 @@
     "loader/"
   ],
   "scripts": {
-    "build": "npm run build:locales && stencil build --docs && npm run build:utils",
+    "build": "npm run build:locales && stencil build && npm run build:utils",
```

Comment:
`--docs` builds the readme as well, since we define the `docs-json` in the config, it generates correctly during the build